tests/zfs-tests/tests/functional/migration/setup.ksh

```diff
@@ -64,10 +45,4 @@ log_must $ZFS set mountpoint=$TESTDIR $TESTPOOL/$TESTFS
 $RM -rf $NONZFS_TESTDIR  || log_unresolved Could not remove $NONZFS_TESTDIR
 $MKDIR -p $NONZFS_TESTDIR || log_unresolved Could not create $NONZFS_TESTDIR
 
-$ECHO "y" | $NEWFS -v ${DEV_DSKDIR}/$NONZFSSIDE_DISK
```

Comment:
@behlendorf  We have two linux which centos 6.1 and customize linux. I want that the migration tests  can be run on my environment, so that only considering the native linux. 
Thanks for your advice, I try to fix it